object-browser.md

```diff
@@ -0,0 +1,68 @@
+```hbs
+{{frost-object-browser-lts
+  items=sortedItems
+  header=(component 'frost-info-bar'
+    icon=(component 'frost-icon'
+      icon='list-medium'
+      pack='frost'
+    )
+    title=(component 'text-box'
+      isVisible=isTitleVisible
+      text='&lt;placeholder: title&gt;'
+    )
+    summary=(component 'text-box'
```

Comment:
`text-box` is the custom component here, but as far as I can tell, `info-bar` only takes component for each API